pkg/backends/locust/resources.go

```diff
@@ -264,7 +265,8 @@ func newWorkerJob(
 			Template: coreV1.PodTemplateSpec{
 				ObjectMeta: metaV1.ObjectMeta{
 					Labels: map[string]string{
-						"app": name,
+						"app":            name,
+						loadTestLabelKey: loadTestWorkerLabelValue,
```

Comment:
this will fail, because `loadTestLabelKey` is equal "app", but there is an existing label with the same key